cores/arduino/Uart.cpp

```diff
@@ -91,13 +91,17 @@ void Uart::flush()
 void Uart::IrqHandler()
 {
   if (sercom->availableDataUART()) {
-    uint8_t c = sercom->readDataUART();
-    if (!sercom->isUARTError()) {
-        rxBuffer.store_char(c);
+
+    if (!sercom->isParityOrFrameError()) {
+        rxBuffer.store_char(sercom->readDataUART());
     } else {
+      sercom->readDataUART();
```

Comment:
I think we should add a comment above this live, saying read and discard the data, as it's invalid